compile.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+RED=$(tput setaf 1)
+RESET=$(tput sgr0)
+
+# Run with `bazel` command as param (build|test)
+if [[ $1 != "build" && $1 != "test" ]]; then
+  echo "$RED""Run script with 'build' or 'test' as param$RESET"
+  exit 1
+fi
+
+# On CircleCI we ignore packages that depend on
+# @com_google_protobuf//:protobuf (cpp library),
+# which leads to inability to use prebuilt binaries
+# and significantly increases build time
```

Comment:
Can you please add a TODO to be able to compile C++ using prebuilt binaries?

Reply:
@oferb will *we* prebuild those deps and include in `startupos_binaries`?

Reply:
Yes